migrations/20170216104317_create_audit_log_for_trial_deduplication.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+exports.up = (knex) => (
+  knex.schema
+    .createTable('trial_deduplication_logs', (table) => {
+      table.uuid('record_id')
+        .primary();
+      table.uuid('trial_id');
+
+      table.text('method')
+        .notNullable();
+    	
+    	table.text('commit')
+    	   .notNullable();
+
+      table.timestamps(true, true);
+
+    })
+    .raw(`ALTER TABLE trial_deduplication_logs ADD CONSTRAINT   
+            trial_deduplication_logs_trials_id_foreign  FOREIGN KEY 
+            (trial_id) REFERENCES trials (id);`)
+    .raw(`ALTER TABLE trial_deduplication_logs ADD CONSTRAINT  
+            trial_deduplication_logs_records_id_foreign FOREIGN KEY 
+            (record_id) REFERENCES records (id);`)
```

Comment:
No need to add the foreign key constraints manually here, as Knex has a helper method for this. Check https://github.com/opentrials/api/blob/master/migrations/20160216120159_create_initial_schema.js#L68-L70